packages/js/admin-e2e-tests/.eslintrc.js

```diff
@@ -0,0 +1,78 @@
+module.exports = {
+	env: {
+		'jest/globals': true,
+	},
+	extends: [ 'plugin:@woocommerce/eslint-plugin/recommended' ],
```

Comment:
Given all our plugins extend from our own `@woocommerce/eslint-plugin`, and most of these rules are the same across all the other packages, I think we should just move most of these rules to our `@woocommerce/eslint-plugin` package, which will remove a lot of duplicate code in our repo.
We probably would want to update the `recommended.js` file with these, given that is what we are using across all our packages.

Also it might be worth revisiting the [@worpress/eslint-plugin](https://github.com/WordPress/gutenberg/blob/%40wordpress/eslint-plugin%4011.1.0/packages/eslint-plugin/configs/recommended.js) rules to see if we have any overlapping rules that aren't necessary anymore. 
Unfortunately eslint doesn't allow for an easy way to figure out if there are duplicate rules.

Reply:
The duplicate rules could be done at a later time, I think updating the `recommended.js` with most of these rules would be better to do before merging this PR.

Reply:
>  I think we should just move most of these rules to our @woocommerce/eslint-plugin package, which will remove a lot of duplicate code in our repo.

This is a good idea! 💯

Reply:
I've moved most of these rules to our `@woocommerce/eslint-plugin` package and updated eslintrc.js for all packages.